snuba/clickhouse/formatter/expression.py

```diff
@@ -39,42 +39,42 @@ class ClickhouseExpressionFormatter(ExpressionVisitor[str]):
     """
 
     def __init__(self, parsing_context: Optional[ParsingContext] = None) -> None:
-        self.__parsing_context = (
```

Comment:
I had to change the double underscores in order for the subclass to work correctly, not sure if there's a different strategy

Reply:
I'm also wondering if I need to remove the other double underscores in the class for the subclass to work. need to test a few more cases